torchrl/envs/libs/vmas.py

```diff
@@ -627,22 +634,35 @@ class VmasEnv(VmasWrapper):
+            https://github.com/proroklab/VectorizedMultiAgentSimulator?tab=readme-ov-file#main-scenarios
 
     Keyword Args:
-        num_envs (int): Number of vectorized simulation environments.
-        device (torch.device, optional): Device for simulation.
-        continuous_actions (bool, optional): Weather to use continuous actions. Defaults to ``True``.
-        max_steps (int, optional): Maximum number of steps in each vectorized environment after which done is returned.
-            Defaults to ``None`` (no truncation).
+        num_envs (int): Number of vectorized simulation environments. VMAS perfroms vectorized simulation using PyTorch.
+            This argument decides the number of vectorized envuironments that should be simulated in a batch. It will also
+            determine the batch size of the environment.
+        device (torch.device, optional): Device for simulation. Defaults to ``"cpu"``. All the tensors created by VMAS
+            will be placed on this device.
+        continuous_actions (bool, optional): Whether to use continuous actions. Defaults to ``True``. If ``False``, actions
+            will be discrete. The number of actions and their size will depend on the scenario chosen.
+            See the VMAS repositiory for more info.
+        max_steps (int, optional): Horizon of the task. Defaults to ``None`` (infinite horizon). Each VMAS scenario can
+            implement a ``done`` function that will define when the scenario is terminated. If ``max_steps`` is specified,
+            the scenario will also be terminated after this horizon has been reached. If instead of terminating the scenario
+            you wish to truncate it, please use a :class:`~torchrl.envs.transforms.StepCounter` transform.
         categorical_actions (bool, optional): if the environment actions are discrete, whether to transform
-            them to categorical or one-hot.
+            them to categorical or one-hot. Defaults to ``True``.
         group_map (MarlGroupMapType or Dict[str, List[str]], optional): how to group agents in tensordicts for
             input/output. By default, if the agent names follow the ``"<name>_<int>"``
             convention, they will be grouped by ``"<name>"``. If they do not follow this convention, they will be all put
             in one group named ``"agents"``.
             Otherwise, a group map can be specified or selected from some premade options.
             See :class:`~torchrl.envs.utils.MarlGroupMapType` for more info.
+        **kwargs (Dict, optional): These are additional argument that can be passed to the VMAS scenario constructor.
+            (e.g., number of agents, reward sparsity). The available arguments will vary based on the scenario chosen.
+            To see the available arguments for a specific scneario, see the constructor in its file from
+             https://github.com/proroklab/VectorizedMultiAgentSimulator/tree/main/vmas/scenarios
```

Comment:
same, let's use a link to a specific commit if possible + \`text \<link\>\`_

Reply:
If I link to a specific commit, users won't see new senarios

Reply:
sure but if the link breaks we won't know about it either. A fixed link seems more conservative to me, wdyt?

Reply:
If there was a doc we could link to the doc which I would assume would not change, but given that there isn't the best seems to be a link to some commit we know won't change

Reply:
ok done